openvdb_cmd/vdb_ax/cli.h

```diff
@@ -0,0 +1,402 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+/// @file cmd/cli.h
+
```

Comment:
can we maybe comment here to say this is not intended to be part of the API or something?

Reply:
Done